dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/environment/EnvironmentExtractionPipeline.scala

```diff
@@ -60,7 +60,13 @@ object EnvironmentExtractionPipeline extends ScioApp[Args] {
       "nov2020_arm_1",
       "nov2020_secondary_arm_1",
       "dec2020_arm_1",
-      "dec2020_secondary_arm_1"
+      "dec2020_secondary_arm_1",
```

Comment:
Same comment about parameterizing these. I'm wondering if we can consume  "start month" + "end month" args and then dynamically create this list from there.